importer.py

```diff
@@ -21,4 +21,37 @@
                              attachment_key='org.digitalpsych.redcap.data',
                              body=records)
     print(count)
+    try:
+        arr = LAMP.Type.get_attachment(researcher['id'], 'org.digitalpsych.redcap.links')["data"]
+    except LAMP.ApiException:
+        continue # Ignore this Researcher
+    parts = []
+    for study in LAMP.Study.all_by_researcher(researcher['id'])['data']:
+        parts+=(p['id'] for p in LAMP.Participant.all_by_study(study['id'])['data'])
+    for p in parts:
+        try:
+            record_id = LAMP.Type.get_attachment(p, 'org.digitalpsych.redcap.id')["data"]
+        except LAMP.ApiException:
+            continue # Ignore this Participant
+        if record_id >= 0:
```

Comment:
How are you going to create the `record_id ←→ Participant.id` mapping in the first place though? Is that a manual process or something we can automate?

Reply:
This will happen in the college study worker and will run automatically.

Reply:
Nice! 👍